glide/blog.md

```diff
@@ -0,0 +1,147 @@
+
+Here's a small example that adds g + c as a key mapping to switch to the calendar tab:
+
+```typescript
+glide.keymaps.set("normal", "gc", async () => {
+  const tab = await glide.tabs.get_first({
+    url: "https://calendar.google.com/*",
+  });
+  assert(tab && tab.id);
+  await browser.tabs.update(tab.id, { active: true });
+}, {
+  description: "[g]o to [c]alendar.google.com",
+});
+```
+
+For more examples, see the [cookbook](https://glide-browser.app/cookbook) or my own [dotfiles](https://github.com/RobertCraigie/dotfiles/tree/main/glide).
+
+The cherry on top is that all of this is built on _top_ of Firefox, meaning that, if you already use Firefox, then your existing extensions and workflows will still work in Glide.
+
+### Modes
+
+Glide borrows the concept of modes from (neo)vim, every key mapping you define will be attached to a specific mode. Glide will then switch between modes automatically but you can always switch modes manually with custom key mappings.
+
+The default mode is `normal`, where most key mappings are defined. When you click or focus on an input element, Glide will automatically switch to `insert` mode, so that key mappings don't interfere with entering text.
+
+If a website doesn't play well with your key mappings you can switch to `ignore` mode by holding Shift and pressing Escape, in this mode the only default key mapping is Shift + Escape to exit ignore mode.
+
+### Navigating
+
+Glide supports a `hint` mode that lets you operate web pages entirely using the keyboard.
+
+Press `f` to enter `hint` mode and Glide will overlay text [labels](https://glide-browser.app/hints#label-generation) over every [hintable](https://glide-browser.app/hints#hintable-elements) element, e.g. links and buttons. Typing the [label](https://glide-browser.app/hints#label-generation) for a hint will then focus and click the element.
+
+See it in action in the [demo video](https://glide-browser.app/#demo-video).
+
+### Personal favourite features
+
+- `gI` focuses the largest visible input element on the page and it feels like magic every time I use it.
+
+- `<space><space>` opens a tab fuzzy finder, which is invaluable for finding that _one_ tab you always lose.
+
+- `<c-i>` and `<c-o>` are also invaluable for navigating previously open tabs.
+
+- `:repl` for quickly testing out config changes.
+
+- [hints](https://glide-browser.app/hints) for when I don't want to reach for my mouse.
+
+- A [which-key](https://github.com/folke/which-key.nvim) inspired UI for reminding you of the many key mappings.
+
+<br>
+
+I've been daily driving Glide for ~6 months now and while I'm biased, I love it. It's still in a very early alpha stage but if you'd like to try it out, you can download it on macOS / Linux [here](https://glide-browser.app/#download). I'd recommend checking out the tutorial with `:tutor` to get some bearings, although it's not fully complete yet.
```

Comment:
Idk why but i love this paragraph

Reply:
hmmm just the vibe is off? just delete it or rewrite you think?

Reply:
no i like it

Reply:
it sounds like robert

Reply:
oh wait what I completely misread your comment wtf

Reply:
I thought you said you didn't like it lmfao